packages/support/src/helpers.php

```diff
@@ -132,10 +132,11 @@ function generate_href_html(?string $url, bool $shouldOpenInNewTab = false, ?boo
         if ($shouldOpenInNewTab) {
             $html .= ' target="_blank"';
         } elseif ($shouldOpenInSpaMode ?? (FilamentView::hasSpaMode($url))) {
-            $html .= ' wire:navigate';
 
             if (FilamentView::hasSpaPrefetching()) {
-                $html .= '.hover';
+                $html .= ' wire:navigate.hover';
+            } else {
+                $html .= ' x-on:click.prevent="() => window.Alpine.navigate(' . "'{$url}'" . ')"';
```

Comment:
I don't think we should change this attribute for every single link, it seems to risky. Maybe just introduce a new `bool` param for `generate_href_html` for `$hasNestedClickEventHandler` which enables this?